docs/source/noise/index.rst

```diff
@@ -0,0 +1,58 @@
+.. _noise_main:
+
+======
+ Noise
+======
+
+.. contents::
+   :depth: 2
+   :local:
+   :backlinks: none
+
+In order to have a realistic challenge with entity resolution, it is essential
+to add noise to the data. Pseudopeople can add two broad categories of noise to
+the datasets it generates:
+
+#. **Row-based noise:** Errors in the inclusion or exclusion of entire rows of
+   data, such as duplication or omission
+#. **Column-based noise:** Errors in individual data entry, such as miswriting
```

Comment:
Can we explain here, or somewhere, that we currently apply this noise independently to each column?

Reply:
Yeah, I was wondering where to do that. I'll try putting something in.

Reply:
I added a paragraph after this with some more explanation. Let me know if you think it looks good.